discovery-provider/src/models.py

```diff
@@ -43,14 +43,17 @@ def validate_field_helper(field, value, model):
     return value
 
 def get_default_value(field, value, model, e):
-    schema = ModelValidator.get_schema_for_field(None, model)
     field_props = ModelValidator.get_properties_for_field(model, field)
-    field_type = field_props['type']
+    field_type = field_props['type'] #this can either be a string or list, need to check both
     default_value = field_props['default']
 
     # if the field is of type object and the default value isn't set in the schema, set to SQL null
     # otherwise JSONB columns get set to string 'null'
-    if field_type == 'object' and not default_value:
+    
+    # if field is defined as a list, need to check if 'object' is in list, else check string
+    if isinstance(field_type, list) and 'object' in field_type and not default_value:
```

Comment:
also wondering, if a default isn't defined, shouldn't the default always be `null()` / is it necessary to do these if/else checks?

Reply:
so this code is only run if the type is object. we need this to determine if we need to set it to sql null.